src/jsifier.js

```diff
@@ -422,6 +422,8 @@ function JSify(data, functionsOnly) {
     // "Final shape that will be created").
     print('// EMSCRIPTEN_END_FUNCS\n');
 
+    if (ASSERTIONS) print('var ASSERTIONS = true;\n');
+    print(read('arrayUtils.js'));
```

Comment:
It is, but it won't do the exporting operation - see the implementation of that method in `modules.js`. It's ok for it to be a no-op in html, but not in js.